weapons-on-back/cl_weapons-on-back.lua

```diff
@@ -40,27 +41,34 @@ local SETTINGS = {
     }
 }
 
+--- Other Variables ---
 local attached_weapons = {}
+local me = GetPlayerPed(-1)

```

Comment:
This should be called in a loop because if the player changes their ped for some reason or if the ped is loaded after this is called then ``me`` won't be set to the correct ped!

Reply:
Done!